### PR TITLE
Corrected the formula in the docstring of FLV in metrics.py

### PR DESCRIPTION
**Pull request by XuHuanHydro**
The old formula in the FLV document string had '%BiasFMS' on the left side , I fixed it to '%BiasFLV'. 
And I added '*' after the '-1' on the right side of the equal.

### CODE REVIEW

File: neuralhydrology/evaluation/metrics.py [hunk 1]
```diff
@@ -473,7 +473,7 @@ def fdc_flv(obs: DataArray, sim: DataArray, l: float = 0.3) -> float:
     r"""Calculate the low flow bias of the flow duration curve [#]_
     
     .. math:: 
-        \%\text{BiasFMS} = -1 \frac{\sum_{l=1}^{L}[\log(Q_{s,l}) - \log(Q_{s,L})] - \sum_{l=1}^{L}[\log(Q_{o,l})
+        \%\text{BiasFLV} = -1 * \frac{\sum_{l=1}^{L}[\log(Q_{s,l}) - \log(Q_{s,L})] - \sum_{l=1}^{L}[\log(Q_{o,l})
```
Comment (gauchm) [inline neuralhydrology/evaluation/metrics.py:476]: nit-pick: could you change the `*` to `\times` for consistency?
```suggestion
        \%\text{BiasFLV} = -1 \times \frac{\sum_{l=1}^{L}[\log(Q_{s,l}) - \log(Q_{s,L})] - \sum_{l=1}^{L}[\log(Q_{o,l})
```

**Review by gauchm — COMMENTED** [review]
Good catch, thanks! Just a small comment for consistency.

File: neuralhydrology/evaluation/metrics.py (hunk 1 above)
Comment (XuHuanHydro) [reply neuralhydrology/evaluation/metrics.py]: Please take a look at the changes and see if they are good. (This is my first pull request on github. I am still new to it.

**Review by gauchm — APPROVED** [review]

**Comment by gauchm** [comment]
Thanks and congrats on your first PR!

**Pull request merged**